Exception/src/main/java/com/onto/javacourse/between0n100/EmptyListException.java

```diff
@@ -0,0 +1,14 @@
+package com.onto.javacourse.between0n100;
+
+/** Exception thrown when attempting to remove an element from an empty list. */
+public class EmptyListException extends RuntimeException {
+
+  /**
+   * Constructs a new EmptyListException with the specified detail message.
+   *
+   * @param message the detail message explaining the cause of the exception
+   */
+  public EmptyListException(String message) {
```

Comment:
It is a good practice to override also at lease the constructor with throwable. Because now you can't chain exceptions